maintainers/maintainer-list.nix

```diff
@@ -16225,4 +16225,10 @@
     github = "ziguana";
     githubId = 45833444;
   };
+  RossComputerGuy = {
+    name = "Tristan Ross";
+    email = "tristan.ross@midstall.com";
+    github = "RossComputerGuy";
+    githubId = 19699320;
+  };
```

Comment:
Quick note since this is buried deep within the manual (maybe we should add it to CONTRIBUTING.md too...), but additions to `maintainers/maintainter-list.nix` are supposed to be in their own commits titled "maintainers: add [name]": https://nixos.org/manual/nixpkgs/unstable/#var-meta-maintainers

Reply:
Should I move it to a different commit?

Reply:
Yes, please do

Reply:
Done